wled00/button.cpp

```diff
@@ -131,21 +131,41 @@ void handleSwitch(uint8_t b)
   }
 }
 
+#define ANALOG_BTN_READ_CYCLE 250   // min time between two analog reading cycles
+#define STRIP_WAIT_TIME 6           // max wait time in case of strip.isUpdating() 
+#define POT_SMOOTHING 0.25          // smoothing factor for raw potentiometer readings
+#define POT_SENSITIVITY 4           // changes below this amount are noise (POT scratching, or ADC noise)
+
 void handleAnalog(uint8_t b)
 {
-  static uint8_t oldRead[WLED_MAX_BUTTONS];
+  static uint8_t oldRead[WLED_MAX_BUTTONS]  = {0};
+  static float filteredReading[WLED_MAX_BUTTONS] = {0.0};
```

Comment:
You my want to add f to float constants otherwise they may get stored as double by compiler.

Reply:
Ah, good point. Will update. Thanks for reviewing.